docs/source/deployment/argo.md

```diff
@@ -1,6 +1,11 @@
-# Deployment with Argo Workflows
+# Argo Workflows (deprecated)
+
+``` {important}
+This page contains legacy documentation that has not been tested against recent Kedro releases.
+```
+
+<div style="color:gray">This page explains how to convert your Kedro pipeline to use [Argo Workflows](https://github.com/argoproj/argo-workflows), an open-source container-native workflow engine for orchestrating parallel jobs on [Kubernetes](https://kubernetes.io/).
```

Comment:
Oh I understand, it was a way of turning the text color of this whole page gray.

I think the "proper" Sphinx way to do this would be to add a `:deprecated:` metadata field on top, and then perform some `conf.py` hackery (which could be an extension) to modify the CSS of only the pages tagged with it. Similar to what `sphinx-opengraph` does: https://github.com/wpilibsuite/sphinxext-opengraph/blob/f510aeae8cd214ff86147047a61db7b8784cd926/sphinxext/opengraph/__init__.py#L41-L48

Reply:
Sure, but I'm not au fait with Sphinx hackery, as you know :)

We could certainly do that...but is using a `div` to colour the text going to be a problem as a simple approach?

Reply:
Comes from the parent theme https://github.com/readthedocs/sphinx_rtd_theme/blob/master/sphinx_rtd_theme/static/css/theme.css we can override it in https://github.com/kedro-org/kedro/blob/main/docs/source/_static/css/theme-overrides.css

Reply:
> is using a div to colour the text going to be a problem as a simple approach?

Maybe we can go for it for now, although the Markdown links would need to be replaced by HTML

Reply:
> the Markdown links would need to be replaced by HTML

They're working fine in the built docs as far as I can tell.??

Reply:
> They're working fine in the built docs as far as I can tell.??

I still see the `[]()` Markdown syntax for links in the HTML docs 🤔

Reply:
In any case, I fixed the "important" admonition color (picked the "warning" one)